activation/post_supervisor.go

```diff
@@ -19,39 +20,16 @@ import (
 
 // DefaultPostServiceConfig returns the default config for post service. These are intended for testing.
 func DefaultPostServiceConfig() PostSupervisorConfig {
-	cfg := PostSupervisorConfig{
-		PostServiceCmd:  "/bin/service",
-		DataDir:         config.DefaultDataDir,
-		NodeAddress:     "http://127.0.0.1:9093",
-		PowDifficulty:   config.DefaultConfig().PowDifficulty,
-		PostServiceMode: "light",
+	return PostSupervisorConfig{
+		PostServiceCmd: "/bin/service",
```

Comment:
TODO: check if this is actually a good default value - on windows it probably has be `./service.exe` or similar.